express-post-json/server.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+
+const grades = {};
+let nextId = 1;
+const app = express();
+
+app.get('/api/grades', (req, res) => {
+  const gradesArray = [];
+  for (const id in grades) {
+    gradesArray.push(grades[id]);
+  }
+  res.json(gradesArray);
+});
+
+app.use(express.json());
```

Comment:
It's best to put this at the top of the file. That way it will apply to all routes.

Reply:
Okay, will change that.  Thanks Robert.